tests/test_in_memory.rs

```diff
@@ -415,6 +425,36 @@ async fn test_findex() -> Result<(), Error<ExampleError>> {
         check_search_result(&doe_search, &doe_keyword, &john_doe_location);
     }
 
+    // Try deleting John Doe from the `doe_keyword`.
+    let mut indexed_value_to_keywords = HashMap::new();
+    indexed_value_to_keywords.insert(
+        IndexedValue::from(john_doe_location.clone()),
+        HashSet::from_iter(vec![doe_keyword.clone()]),
+    );
+    findex
+        .upsert(
+            HashMap::new(),
+            indexed_value_to_keywords,
+            &master_key,
+            &new_label,
+        )
+        .await?;
```

Comment:
Maybe rename `indexed_value_to_keywords` to `indexed_value_to_keywords_removal`?

Reply:
I changed to `deletions`.